akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -884,7 +884,7 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
     OneForOneStrategy()(
       ({
         case e @ (_: DurableStore.LoadFailed | _: ActorInitializationException) if fromDurableStore ⇒
-          log.error(e, "Stopping distributed-data Replicator due to load or startup failure in durable store")
+          log.error(e.getCause, "Stopping distributed-data Replicator due to load or startup failure in durable store")
```

Comment:
Ah, yes, you are right. The reason was the missing cause in LoadFailed. After that it will automatically be shown in the stacktrace. I copied your suggesting for the string. That's nice.